packages/mdc-fab/README.md

```diff
@@ -121,28 +157,6 @@ You can also do this declaratively when using the [material-components-web](../m
 
 MDC FAB is fully aware of MDC Ripple styles, so no DOM or CSS changes are required.
 
-### Sass Mixins
-
-By default an MDC FAB will inherit its color from the theme. This mixin will override the color of the MDC FAB's container, but maintain accessibility standards for the ink and ripple. The mixin is intended for customizing an MDC FAB's color to a non-theme color.
-
-#### `mdc-fab-accessible($container-color)`
-
-Changes the FAB's container color to the given color, and updates the FAB's ink and ripple color to meet accessibility standards.
-
-### Advanced Sass Mixins
-
-> **A note about advanced mixins**, The following mixins are intended for advanced users. These mixins will override the color of the container, ink, or ripple. You can use all of them if you want to completely customize a FAB. Or you can use only one of them, e.g. if you only need to override the ripple color. **It is up to you to pick container, ink, and ripple colors that work together, and meet accessibility standards.**
-
-Mixin | Description
---- | ---
-`mdc-fab-container-color($color)` | Sets the container color to the given color
-`mdc-fab-icon-size($width, $height)` | Sets the icon `width`, `height`, and `font-size` properties to the specified `width` and `height`. `$height` is optional and will default to `$width` if omitted. The `font-size` will be set to the provided `$width` value.
-`mdc-fab-ink-color($color)` | Sets the ink color to the given color
-
-The ripple effect for the FAB component is styled using [MDC Ripple](../mdc-ripple) mixins.
-
-#### Caveat: Edge and CSS Variables
+## Usage within Web Frameworks
```

Comment:
This section can be omitted because FAB has no javascript. (Also, it says Select...)